### ISSUE TITLE
make start fails

### ISSUE THREAD

**Issue by zeeMonkeez** [SwissDataScienceCenter/renga#110]
Following instructions at http://renga.readthedocs.io/en/latest/user/setup.html#setup. Calling `make start` fails:

> ...
> srcrenga_explorer_1 is up-to-date
> srcrenga_storage_1 is up-to-date
> Starting srcrenga_graph-init_1 ... done
> docker: Error response from daemon: No such network: renga_default.
> make: *** [start] Error 125

Not clear to me how to solve this.

Running on macOS 10.12. Docker 17.09.0-ce-mac35 (19611) is installed, memory increased as recommended.

**Comment by rokroskar** [comment]
hi @zeeMonkeez thanks for the report -- unfortunately I can't reproduce this problem. It looks like it could be an issue with docker or perhaps a mismatch between docker and docker-compose versions. Have you tried restarting your docker? And could you post the output of 

```
docker-compose --version
```

**Comment by zeeMonkeez** [comment]
I installed docker from the official download page, at https://download.docker.com/mac/stable/Docker.dmg which to my understanding includes `docker-compose`.  
`docker-compose --version` returns

> `docker-compose version 1.16.1, build 6d1ac21`

Tried restarting docker.

It seems the string `renga_default` only occurs in the script `scripts/wait-for-services.sh`. Is it possible that there is a docker configuration step that has been omitted from the documentation so far?
